eng/mgmt/mgmtmetadata/web_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/web/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=E:\github\azure-sdk-for-net\sdk
-2020-06-05 06:44:53 UTC
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/web/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=K:\DotNet-SDK\Sep-2020\azure-sdk-for-net\sdk
```

Comment:
It seems you were still generating with the default tag. Could you check in your `generate.ps1` as well?

Here's my `generate.ps1` file for reference
```powershell
Start-AutoRestCodeGeneration -ResourceProvider "web/resource-manager" -AutoRestVersion "v2" -ConfigFileTag package-2020-06
```

Reply:
@isra-fel thanks for your help. I did updated my generate.ps1 but some how it got reverted. Now it is generated properly. Thanks once again.